examples/simple/README.md

```diff
@@ -0,0 +1,6 @@
+
+# Simple Trace Example
+
+In this example, an application in `main.cc` initializes and registers a tracer provider from the <a href="https://github.com/open-telemetry/opentelemetry-cpp"> Open Telemetry SDK </a>.  The application then calls a `foo_library` which has been manually instrumented using the <a href="https://github.com/open-telemetry/opentelemetry-cpp/tree/master/api"> OpenTelemetry API </a>.  Resulting telemetry is directed to stdout through a custom exporter.
```

Comment:
Remove the extra space before/after `<a>`/`</a>`. Otherwise it'll render an extra white space which is buggy.

![image](https://user-images.githubusercontent.com/17327289/84345263-c5630600-ab61-11ea-9551-a64c08b53413.png)

Reply:
Please use [markdown links](https://www.markdownguide.org/basic-syntax/#links).

```
[Open Telemetry SDK](https://github.com/open-telemetry/opentelemetry-cpp)
```